js/bybit.js

```diff
@@ -2415,8 +2490,14 @@ module.exports = class bybit extends Exchange {
         let lastTradeTimestamp = this.safeTimestamp (order, 'last_exec_time');
         if (lastTradeTimestamp === 0) {
             lastTradeTimestamp = undefined;
+        } else if (lastTradeTimestamp === undefined) {
+            lastTradeTimestamp = this.parse8601 (this.safeString2 (order, 'updated_time', 'updated_at'));
+            if (lastTradeTimestamp === undefined) {
+                lastTradeTimestamp = this.safeNumber (order, 'updateTime');
+            }
         }
-        const status = this.parseOrderStatus (this.safeString2 (order, 'order_status', 'stop_order_status'));
+        const raw_status = this.safeStringLowerN (order, [ 'order_status', 'stop_order_status', 'status', 'orderStatus' ]);
```

Comment:
I would suggest that we don't lowercase it here, even at a cost of some duplication in `parseOrderStatus`. I mean, we better keep the original source string untouched in `parseOrderStatus`.